cmd/kubeops/internal/handler/handler_test.go

```diff
@@ -147,19 +155,12 @@ func TestActions(t *testing.T) {
 			if response.Code != test.StatusCode {
 				t.Errorf("Expecting a StatusCode %d, received %d", test.StatusCode, response.Code)
 			}
-			releases := derefReleases(cfg.ActionConfig.Releases)
-			rlsComparer := cmp.Comparer(func(x release.Release, y release.Release) bool {
-				return x.Name == y.Name &&
-					x.Version == y.Version &&
-					x.Namespace == y.Namespace &&
-					x.Info.Status == y.Info.Status &&
-					x.Chart.Name() == y.Chart.Name() &&
-					x.Manifest == y.Manifest &&
-					cmp.Equal(x.Config, y.Config) &&
-					cmp.Equal(x.Hooks, y.Hooks)
-			})
-			if !cmp.Equal(releases, test.RemainingReleases, rlsComparer) {
-				t.Errorf("Unexpected remaining releases. Diff %s", cmp.Diff(releases, test.RemainingReleases, rlsComparer))
+			releases, err := cfg.ActionConfig.Releases.ListReleases()
+			if err != nil {
+				t.Fatalf("%+v", err)
+			}
+			if !cmp.Equal(releases, test.RemainingReleases, releaseComparer) {
+				t.Errorf("Unexpected remaining releases. Diff %s", cmp.Diff(releases, test.RemainingReleases, releaseComparer))
```

Comment:
I haven't run the code, but wouldn't something like this be nice?
```suggestion
				t.Errorf("Unexpected remaining releases. Diff:\n%s", cmp.Diff(releases, test.RemainingReleases, releaseComparer))
```